### DISCUSSION TITLE
AppStore error : ITMS-91061: Missing privacy manifest

### DISCUSSION THREAD

**Discussion by shayan-izadi** [timbru31/cordova-plugin-lottie-splashscreen#845] [General]
Hi. I am using the latest version of this plugin but when I submit the app in Appstore I get this error from them:

ITMS-91061: Missing privacy manifest - Your app includes “Frameworks/Lottie.framework/Lottie”, which includes Lottie, an SDK that was identified in the documentation as a privacy-impacting third-party SDK. Starting November 12, 2024, if a new app includes a privacy-impacting SDK, or an app update adds a new privacy-impacting SDK, the SDK must include a privacy manifest file. Please contact the provider of the SDK that includes this file to get an updated SDK version with a privacy manifest. For more details about this policy, including a list of SDKs that are required to include signatures and manifests, visit: https://developer.apple.com/support/third-party-SDK-requirements.

Apple Developer Relations

**Comment by Binuka97** [comment]
Any update for this?
